test/corruption.test.js

```diff
@@ -0,0 +1,166 @@
+"ToMU0jPheW/sUBfVJvnOWxKpclzspBWA1wqAouIIriCSCUc0UrS144FDNh787NyNyDpKLqjZ5raW68IL" +
+"aIevPh+Vr6bW27yzm/6Tg5dbkl5Vjyqh0Oc344qEr0PupKm5/cuh1OdsuAdq6s5VSLwn4qlC7iPfmDvS" +
+"g4il0/FKByKI+8uJo7eEc00z2xxtGsueeUDzlG0lVlFVui2necsxttY2Vi017hjW+SNoYF4p68xSatw2" +
+"tyl8F8T0TKrdE+RJESvNyYCA8JV8UDLwTefMW/2Od/maWf8AcXb+jbf/AKZPdB++JH5i/oXOb4F6SQwQ" +
+"BAePFWEbwgOEdbWhs+qL1tKNmcJmcRIOZx+PmUNjIUnznPZhCl1veRLHVWmzQZWrSgQGz9E5mC3nlxl5" +
+"Q2d96oDvZ5yOUtPB40KQwGI4JUe33kll2IUW4S8MvbtNG636VvOmck4Brn4udxNncawBr7t52Pb6da73" +
+"CYpXY/uWs0sZg5WZfteruNfbecVtmnUq+c4oKlDrzRrQVOmeFXSE3et6myrO6togXY6OQULnHR3xB90e" +
+"5vOlQua4+FuDVaJaZPckTeVYGTkpyX9Pebpe3Rubh0vu6mDcBqXz1nWZPGYmV37dUf6Vq7z0DD2fLgkY" +
+"6ijMFUFD3UC2LUAS3SUBkyEk1NETA0Hi81cPJyN869H9H4ekJ3N7S6tPxInMp6Ujc12ZGBAEAQHMfFzD" +
+"u5bfKRt/TPczEfVcasPkdUfeWljbdY13EbmVqsVLcc6ik0KJaIJovgqwtPVQBAbTi+q7WeyOK6ghF5ZP" +
+"HIZHt5/V2B7dtNjhpUphcwcGuLZtJSxj048F1Vj7ayLv/CHBZIm46cy/cNdpFvJ+ewcAQWyN+9VdNYzi" +
+"q00lzFZZbbuabU/j8yOj8DOoi+kmWtGx7XNjlcadhDfpWy81h+lmJZPc/VHtJvH9DdBdLFt3mr3+qXsf" +
+"rMheByBw+rAwur980Udis3dKV4ebWbEMJYsaZvil7bO8mLPPXuflddmM22LhPLaW+17hoMjyPq6mgaAv" +
+"MPV2cuSWHhorpl8F8eo6DJ4yvN3WqQWiPxfwRnLgDogqg8JoskIgx5JBtNBtJ2KQs2yhufSdk6DHiR45" +
+"ZJj3jgdY5qcoPENABXr2V4T+Ph429tNPO9LOdxFzjm2TikDCEAQBAR+dxcGSxs9pM3mjlYWupr07RxGs" +
+"KjVVRlJRTVGfP2SsLnFZKaxuRSSF1K7HN1tcOBGlQl604uhzV+y4ScWeRyLXaNZouhytoWlSA9VAAS08" +
+"zSQd40FVBVNeXZj5TPIQdhe6n0q5Te8v45b2YlpYy317HaxaHSn1nbm63OPYFhxeKjh7UrktS7XsRsYT" +
+"DSv3FCO32qdJtraG2t47eEcsUTQ1g4BeWX70rs3OXik6npdm1G3BQjqRcWJIyFJdRZYwBZkkW7atFC9i" +
+"LF2Rv2x0rCyjpt1K6G/eI81V13pzLfMu+ZJfRDtls6tfUaWOv8MeFa37josUYjjDRsXoRCFaAIAgCAID" +
+"RfEXov8Aq1r83aNAv4Ae7OrnbrMZP93ce0rBfs8a5TVxWH8yP7kcbD5IpHRStLJGEtexwoQQaEEFREoU" +
+"OflFp0esyWTVWJoxtF5sgVtC2hUHBUoUoe1QFuV2zcqpFTauk8X8vam8lFJrgepXWI9nxa1xHqLHebc8" +
+"qPhhr5ZfLV1ncen8B5dvzZeKer+n59xPFwXOq2dCUukCzwtAsyTALbt2ShZjbNdTtggbzyP9kbNGsk7A" +
+"Nv76KZy/Lp35qMel7kYr15W41Z0HAYePH2obrkd6z30oXOOs/wBti9LwuGhYtqENSOfuXHOVWSq2CwIA" +
+"gCAIAgPHNa5paRUFAc+688OYspzX1jSG/A0n3ZKbH027nefhgvWFPnNTE4RXNK0SOQ3UN7j7p1rexOgn" +
+"Z7THinlGwjiFGTtOLoyDuW5QdJIqjuQdqwuJioX2zjereEpwlYmG9U4SlDLwtl/UL8Md+gz15j9kbPKt" +
+"HMsV5FpteJ6F39BJZVgP5F5J+BaZd3Sb13rQKDQBqAXAqzU9EKHTjessbALMlyBtWzCwDy1t7u/m7m3Y" +
+"XH3ne62v1j+zWprL8puX3oVI7/bWYL+IjbWnXuN86f6cgx8XM715nU55CNJp9AGwft0rvcHg7eHhwwXz" +
+"IO9elcdWTq2jEEAQBAEAQBAEAIBFDqQED1D0fiM1bmO7ga/XyO1OaTta4aQrZRUlRlly3GapJVOWZ7wk" +
+"zFm50mLlFzFrEMlGSDsd7DvwrUnhNxGXcteuD6zTb2xzGOdy31pLb0957SG+R3snyFa0rElrRoTsTjrT" +
+"McXhI0aa6BRYuAxG9YOD5Cxaxw/Pk9eY8Tqb5FyGYXPPuV+1aF7cp6BlWB/j2Un45aX3dBnPvOUVceUb" +
+"zoHpWpHD11Ekyu3gv7sj5eB8gOpwFG/G6jfSpPD5Nfuao0XLoNeeKtx1vqNgxfRN1MQ+9fyt/wBOMn0v" +
+"ND5gO1dDhPT9uGm4+J7tnzNC7mEnojoNzx+JtLKJscMYaG6gBQKfjFRVEqIj229LM1XFAgCAIAgCAIAg" +
+"CAIAgPHNa4UIqgMabG2soPMwadaAibjofp6d/PJYwOfWoeYmc1d9aVVsop60U4VWtCkdDYMGvy0fwhYf" +
+"4ln9EepGXzp731mVbdK4i3dzRW8bHfWaxrT5wFmjCMdSoWOTeskYrK3j9lgrvVxQvgAakAQBAEAQBAEA" +
+"QBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAf/9k=";
```

Comment:
Actually, reading again it seems like this file isn't corrupt in of its self. Could you use the already existing `checkers.png` or `clock.png`?